hibernate-core/src/test/java/org/hibernate/orm/test/sql/exec/onetoone/OneToOneWithDerivedIdentityTest.java

```diff
@@ -52,7 +52,7 @@ public void testGet() {
 		inTransaction(
 				session -> {
 					Person person = session.get( Person.class, PERSON_ID );
-					assertEquals( person.getName(), "Alfio" );
+					assertEquals( "Alfio", person.getName() );
```

Comment:
A common erroneous usage of `assertEquals` in our codebase.

Reply:
Yes, this is once of the great things about hamcrest which we use in newer tests